Mathlib/Data/List/Basic.lean

```diff
@@ -197,6 +197,14 @@ theorem append_right_injective (s : List α) : Injective fun t ↦ s ++ t :=
 theorem append_left_injective (t : List α) : Injective fun s ↦ s ++ t :=
   fun _ _ ↦ append_cancel_right
 
+theorem append_cancel_right_length {as bs bs' cs : List α}
+(eq_length : bs.length = bs'.length) (h : as ++ bs = cs ++ bs') : as = cs := by
```

Comment:
```suggestion
    (eq_length : bs.length = bs'.length) (h : as ++ bs = cs ++ bs') : as = cs := by
```
([the convention is to indent four spaces here](https://leanprover-community.github.io/contribute/style.html))